core/lib/include/assert.h

```diff
@@ -22,6 +22,8 @@
 #ifndef ASSERT_H
 #define ASSERT_H
 
+#include "panic.h"
```

Comment:
please do not in include any other header in libc lookalike header libc (by default)

Reply:
http://www.open-std.org/jtc1/sc22/wg14/www/C99RationaleV5.10.pdf :7.1.3 p 112 (pdf:119) "...the Standard assures that macro and typedef names are reserved only if the associated header is explicitly included"

"Also reserved for the implementor are all external identifiers beginning with an underscore, and all other identifiers beginning with an underscore followed by a capital letter or an underscore." (same doc same §)

these buildins kind of things of gcc are __ : __attribute__ __LINE__
glibc assert.h uses __assert_fail

see dissicusion in  #17574